states.cpp

```diff
@@ -0,0 +1,492 @@
+#include "wl_def.h"
+#include "wl_def.h"
+
+extern void A_Breathing (objtype *);
+extern void A_Dormant (objtype *);
+extern void A_HitlerMorph (objtype *);
+extern void A_MechaSound (objtype *);
+extern void A_Relaunch (objtype *);
+extern void A_Slurpie (objtype *);
+extern void A_Smoke (objtype *);
+extern void A_StartAttack (objtype *);
+extern void A_StartDeathCam (objtype *);
+extern void A_Victory (objtype *);
+extern void T_Attack (objtype *);
+extern void T_BJDone (objtype *);
+extern void T_BJJump (objtype *);
+extern void T_BJRun (objtype *);
+extern void T_BJYell (objtype *);
+extern void T_Bite (objtype *);
+extern void T_Chase (objtype *);
+extern void T_DogChase (objtype *);
+extern void T_Fake (objtype *);
+extern void T_FakeFire (objtype *);
+extern void T_Fat (objtype *);
+extern void T_Ghosts (objtype *);
+extern void T_Gift (objtype *);
+extern void T_GiftThrow (objtype *);
+extern void T_Launch (objtype *);
+extern void T_Path (objtype *);
+extern void T_Player (objtype *);
+extern void T_Projectile (objtype *);
+extern void T_Schabb (objtype *);
+extern void T_SchabbThrow (objtype *);
+extern void T_Shoot (objtype *);
+extern void T_Stand (objtype *);
+extern void T_UShoot (objtype *);
+extern void T_Will (objtype *);
+
+statetype states[numstates] = {

```

Comment:
const?

Reply:
For the moment it is `const`, yes. But once I will tackle #13, sprite numbers will have to get shuffled around.